controllers/articles.controllers.js

```diff
@@ -22,3 +22,19 @@ exports.getArticleById = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.updateArticleVotesById = (req, res, next) => {
+  const { article_id } = req.params;
+  const { inc_votes } = req.body;
+
+  const promises = [
+    selectArticleById(article_id),
```

Comment:
This select shouldn't be needed. If you try to update an article which doesn't exist, you should get an SQL error 🙂